http-device/src/main/java/io/strimzi/streams/HttpDevice.java

```diff
@@ -70,7 +70,7 @@ public void stop(Future<Void> stopFuture) throws Exception {
         Future<List<OffsetRecordSent>> fut = Future.future();        
 
         //TODO: better sensor simulation
-        int temperature = 20 + random.nextInt(5);
+        int temperature = config.getMinTemperature() + random.nextInt((config.getMaxTemperature() + 1 - config.getMinTemperature()));
```

Comment:
Why the +1. Cannot happen to have 26 as value in that case so higher than the max 25?

Reply:
`nextInt`
Returns a random number.
between 0 (inclusive) and n (exclusive).

Reply:
Right. Si just remove the TODO comment and we are set.